omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1087,34 +1087,36 @@ def get_crop_region(self, panel):
 
         return {'x': cropx, 'y': cropy, 'width': tile_w, 'height': tile_h}
 
-    def get_time_label_text(self, delta_t, format):
+    def get_time_label_text(self, delta_t, format, dec_prec):
```

Comment:
A default value should be given to ``dec_prec`` e.g. ``dec_prec = -1`` so we do not change the behaviour of the method